packages/smart/tasks/setTrustedResolution.ts

```diff
@@ -0,0 +1,22 @@
+import { task, types } from "hardhat/config";
+
+import "hardhat/types/config";
+import { buildContractInterfaces, ContractInterfaces } from "..";
+
+task("setTrustedResolution", "Set turbo resolution for the TrustedArbiter")
+  .addParam("turbo", undefined, undefined, types.int)
+  .addParam("outcomes", undefined, undefined, types.json)
+  .setAction(async ({ turbo, outcomes }, hre) => {
+    if (typeof turbo !== "number") return;
+    if (!Array.isArray(outcomes))
+      throw Error(`Outcomes must be an array of strings that represent numbers, not ${outcomes}`);
+    const { ethers } = hre;
+
+    const [signer] = await ethers.getSigners();
```

Comment:
It needs to get moved out of deploy.ts, but makeSigner(...) is actually better. It produces a signer that works for L2 and can move faster, with less chance of random failure.